examples/python/rgbd/main.py

```diff
@@ -91,12 +79,12 @@ def log_nyud_data(recording_path: Path, subset_idx: int = 0) -> None:
                 img_depth = read_depth_image(buf)
 
                 # Log the camera transforms:
-                rr.log_view_coordinates("world/camera", xyz="RDF")  # X=Right, Y=Down, Z=Forward
                 rr.log_pinhole(
                     "world/camera/image",
-                    child_from_parent=camera_intrinsics(img_depth),
                     width=img_depth.shape[1],
                     height=img_depth.shape[0],
+                    focal_length_px=0.7 * img_depth.shape[1],
+                    camera_xyz="FUR",
```

Comment:
I think the choice of FUR here warrants a comment. I'm having a very hard time wrapping my head around what FUR means in this context in an intuitive fashion.  The fact that "depth points forward" checks out, but which way do pixel-x / pixel-y align with U and R?

Reply:
I'll remove it. No need to encourage weird non-RDF usage when it is not warranted